src/Components/ADT3DSceneBuilder/ADT3DSceneBuilder.BehaviorForm.stories.tsx

```diff
@@ -173,77 +173,77 @@ EditTwinAliasesTabAddAlias.play = async ({ canvasElement }) => {
     await userEvent.click(addTwinAliasButton);
 };
 
-export const EditStatusTab = Template.bind({});
-EditStatusTab.play = async ({ canvasElement }) => {
-    await EditElementsTabSelectItem.play({ canvasElement });
-    const canvas = within(canvasElement);
-    // Finds the tabs and clicks Status
-    const tab = await canvas.findAllByRole('tab');
-    await userEvent.click(tab[2]);
-};
+// export const EditStatusTab = Template.bind({});
```

Comment:
do any of these scenarios make sense to wire up against the visual rules logic so we still have them covered?